packages/react-table/src/components/Table/utils/decorators/selectable.tsx

```diff
@@ -42,8 +42,8 @@ export const selectable: ITransform = (
           'aria-labelledby': rowLabeledBy + rowIndex
         }
       : {
-          checked: allRowsSelected,
-          'aria-label': 'Select all rows'
+          'aria-label': 'Select all rows',
+          'data-rowsamount': selectedRowsAmount
```

Comment:
Could we use `data-selectedrowsamount` (or something similar) instead so the markup is more representative of what's going on behind the scenes?